WcaOnRails/db/migrate/20180912042457_add_confirmed_at_to_competitions.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class AddConfirmedAtToCompetitions < ActiveRecord::Migration[5.2]
+  def up
+    add_column :Competitions, :confirmed_at, :datetime
+    execute <<-SQL
+      UPDATE Competitions
+      SET confirmed_at = announced_at
+      WHERE announced_at = 1
```

Comment:
Hmm, shouldn't this be `WHERE announced_at IS NOT NULL`?

Reply:
Oh whoops, you're totally right! Good catch.